contrib_bots/lib/translate.py

```diff
@@ -0,0 +1,95 @@
+# See zulip/contrib_bots/lib/readme.md for instructions on running this code.
+from __future__ import print_function
+import sys
+import logging
+try:
+    from mstranslator import Translator
+except ImportError:
+    logging.error("Dependency Missing!")
+    sys.exit(1)
+
+#Uses mstranslator module
+#  pip install mstranslator
+
+translator = Translator('8f71c4d183fe45cca6fd91cdf73f12ea')
```

Comment:
Take a look at [CommuteBot](https://github.com/zulip/zulip/tree/master/contrib_bots/bots/commute_bot), for instance, and see how it manages API keys through an external file. Feel free to ask anything you find confusing in that code :)